src/DashJS/SDK/Platform/methods/contracts/create.ts

```diff
@@ -1,5 +1,13 @@
 import {Platform} from "../../Platform";
 
+/**
+ * Create contracts on the platform
```

Comment:
Minor suggestion - This might be more accurate since nothing actually goes to the platform until the `broadcast`.
```suggestion
 * Create and prepare contracts for the platform
```

Reply:
Also fixed in https://github.com/dashevo/DashJS/pull/31